### DISCUSSION TITLE
Upgrading LLVM Process

### DISCUSSION THREAD

**Discussion by hamptonm1** [openxla/community#105] [stablehlo]
Is there a specific process that this community follows to upgrade LLVM? I noticed it is done quite frequently which is wonderful but I just wanted to understand the process of selecting the LLVM commit or if there is anything documented about it. Thanks!!

**Comment by GleasonK** [comment]
For StableHLO, we aim to integrate the latest StableHLO changes into `openxla/xla` twice a week, which involves bumping the revision of LLVM we use in StableHLO to match what openxla/xla is using.

The commit used in openxla/xla is roughly updated "as often as possible" AFAIK - bumping the revision and fixing in all google-LLVM-dependent repos (TF included) can take some time if there are breaking upstream changes. As for what specific commit is picked, I'll let someone else with more visibility into the matter chime in - maybe @jpienaar has some insights?

**Reply by hamptonm1** [reply]
Thank you for your response! This information is proving to be helpful :)
